test/extended/openstack/kuryr.go

```diff
@@ -0,0 +1,153 @@
+package openstack
+
+import (
+	"context"
+	"fmt"
+	"math/rand"
+	"strconv"
+	"time"
+
+	"github.com/gophercloud/gophercloud"
+	"github.com/gophercloud/gophercloud/openstack/networking/v2/subnets"
```

Comment:
You need to include the subnets dependency to the vendor folder. Currently, only these resources are available https://github.com/openshift/openstack-test/tree/main/vendor/github.com/gophercloud/gophercloud/openstack/networking/v2.
`go mod vendor` will help you with it.